### DISCUSSION TITLE
DAIKIN three-phase sensors

### DISCUSSION THREAD

**Discussion by oraello** [raomin/ESPAltherma#178] [Q&A]
Hi all,
first of all, many thanks to @raomin for this great project.
I have some problem to get the right sensors to calculate the power and then also the COP.
I have a Daikin Altherna 3 H/HT, ETVX16S23DA9W/EPRA18DAW1 that is three-phase - 400 Volts powerd.
I get values from {0x21,2,101,2,1,"Voltage (N-phase) (V)"}, and {0x21,0,105,2,1,"INV primary current (A)"}, but the power calculated in this way is too low, so possibily this sensor can be related only to 1 phase.
Any idea how to get the right values?
The backup solution of course can be to use an external power meter, but I would like to solve everything with internal sensors.

Enrico

**Comment by TRIROG** [comment]
Did you solve this?

**Reply by oraello** [reply]
No, now using an external power meter

**Comment by mx-vatte** [comment]
My primary current (A) value is also 1/3 from the expected and compared value with a power meter.
As solution I simply multiply the primary current (A) value by 3.

@oraello can you confirm the result multiplied by 3 "equals" your external power meter reading?

**Comment by oraello** [comment]
Could be. I will check as soon as possible, I have to reconfigure sensor ad
move the power meter on that line (now is measuring the entire boiler room).


Il giorno gio 22 giu 2023 alle ore 11:13 mx-vatte ***@***.***>
ha scritto:

**Comment by theailer** [comment]
My N-phase voltage is all over the place.
My pump is a EBLA11D3W1, which is a three phase mono block and I had to add the register to the definition file so maybe that's the issue.

Should it look like that, I'm guessing the N-Phase voltage should be the voltage between neutral and phase which is like 230V?
![image](https://github.com/raomin/ESPAltherma/assets/5675706/08c85d39-25da-4193-907d-901956aac4e7)

**Comment by theailer** [comment]
I read through the discussions here and it seems that N-Phase voltage is the voltage after the transformer in the heatpump.
And INV secondary current (A) is current after the transformer. So in my simple mind we should just change the primary to the secondary current?

**Comment by TRIROG** [comment]
Ha? So which do we need?
